### PR TITLE
loupe: Initial inclusion 

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Initial inclusion of loupe
- Resolves https://github.com/getsolus/packages/issues/1272

**Test Plan**

Open various image files (jpg, png, svg, webp)

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/g/glycin-loaders/package.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name       : glycin-loaders
```
Comment (ReillyBrogan) [inline packages/g/glycin-loaders/package.yml:1]: I feel like `glycin` is a better name.

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

File: packages/g/glycin-loaders/package.yml (hunk 1 above)
Comment (malfisya) [reply packages/g/glycin-loaders/package.yml:1]: I matched it with source name and fedora named it this too.  `glycin` is definitely shorter. :smile:

File: packages/g/glycin/package.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+description: |
+    Glycin allows to decode images into gdk::Texture(s) and to extract image metadata. The decoding happens in sandboxed modular image loaders.
+builddeps  :
+    - pkgconfig(cairo)
+    - pkgconfig(gtk4)
+    - pkgconfig(libheif)
+    - pkgconfig(libxml-2.0)
+    - cargo
+    - git
+    - rust
```
Comment (ReillyBrogan) [inline packages/g/glycin/package.yml:21]: `rust` is pulled in by `cargo` and so you don't need to specify it manually

File: packages/g/glycin/package.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+setup      : |
+    %meson_configure -Dtest_skip_install=true
+build      : |
+    %ninja_build
+install    : |
+    %ninja_install
+check      : |
+    # It always failed
+    # %ninja_check
```
Comment (ReillyBrogan) [inline packages/g/glycin/package.yml:30]: Do they succeed if you build them outside of solbuild?

File: packages/l/loupe/package.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    - Accessible presentation of the most important metadata.
+builddeps  :
+    - pkgconfig(gtk4)
+    - pkgconfig(gweather4)
+    - pkgconfig(lcms2)
+    - pkgconfig(libadwaita-1)
+    - cargo
+    - desktop-file-utils
+    - itstool
+    - rust
```
Comment (ReillyBrogan) [inline packages/l/loupe/package.yml:24]: Same here with having an unnecessary `rust`

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

File: packages/g/glycin/package.yml (hunk 2 above)
Comment (malfisya) [reply packages/g/glycin/package.yml:21]: will remove

File: packages/g/glycin/package.yml (hunk 3 above)
Comment (malfisya) [reply packages/g/glycin/package.yml:30]: Yes, test was ran successfully outside solbuild

File: packages/l/loupe/package.yml (hunk 4 above)
Comment (malfisya) [reply packages/l/loupe/package.yml:24]: ok

File: packages/g/glycin/package.yml (hunk 3 above)
Comment (malfisya) [reply packages/g/glycin/package.yml:30]: That is log of test inside  and outside solbuild.
[inside.txt](https://github.com/getsolus/packages/files/13944225/inside.txt)
[outside.txt](https://github.com/getsolus/packages/files/13944223/outside.txt)

Comment (joebonrichie) [reply packages/g/glycin/package.yml:29]: Maybe you need libpng installed inside solbuild?
`called `Result::unwrap()` on an `Err` value: UnknownImageFormat("image/png")`

Comment (malfisya) [reply packages/g/glycin/package.yml:29]: libpng is deps of libcairo

**Review by ReillyBrogan — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**